.github/workflows/meson.yml

```diff
@@ -116,7 +115,7 @@ jobs:
           if-no-files-found: error
 
   build-macos:
-    runs-on: macos-latest
+    runs-on: macos-11
```

Comment:
Btw if you need to target even older macos versions you might want to set the xcode version to something lower see (seems to go down to catalina):
https://github.com/actions/runner-images/blob/main/images/macos/macos-11-Readme.md#installed-sdks
https://gist.github.com/fabianfett/fd811d7921eb856bb100c5c15565077f

Reply:
Resolved in 521197bb846b959451f1996fe9cf7f789b7dfe8f, Thank you for the input!!! <3